docs/developer-guide.md

```diff
@@ -26,6 +26,30 @@ In this debug mode, `DotnetRunner` does not launch the .NET application, but wai
 
 Now you can run your .NET application with any debugger to debug your application.
```

Comment:
What are the debuggers you are referring to? VS, I understand. Did we also verify VS Code C# debugging?

Reply:
This is only for VS. VS code can be done in a separate PR if needed.

Reply:
I will add VS info